build.gradle

```diff
@@ -181,6 +191,8 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-json', version: versions.springBoot
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: versions.springBoot
 
+  compile group: 'com.github.everit-org.json-schema', name: 'org.everit.json.schema', version: '1.9.1'
```

Comment:
https://github.com/everit-org/json-schema#when-to-use-this-library says if we are using Jackson then better to use https://github.com/java-json-tools/json-schema-validator ?

Reply:
Must've missed that note. will have to convert to that one